pandas/util/testing.py

```diff
@@ -423,32 +421,6 @@ def close(fignum=None):
         _close(fignum)
 
 
-# -----------------------------------------------------------------------------
-# Stdout / stderr decorators
-
-
-@contextmanager
-def set_defaultencoding(encoding):
```

Comment:
@jbrockmendel is this needed in _config/localization.py?

Reply:
I don’t think so, especially not if it’s py2 only.